olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/CatalogClient.h

```diff
@@ -126,7 +140,7 @@ class DATASERVICE_READ_API PrefetchTileResult
       : client::ApiResponse<PrefetchTileNoError, client::ApiError>(error) {}
 
  public:
-  geo::TileKey tile_key_;
+  geo::TileKey tile_key_;///< key of the tile to be pre-fetched
```

Comment:
```suggestion
  /// Key of the tile to be pre-fetched.
  geo::TileKey tile_key_;
```

Reply:
Didn't we discussed that keeping short comments in the same line as the code?

Reply:
Not that I remember. At least until now we only used ///.